builds/securing-builds-by-strategy.adoc

```diff
@@ -0,0 +1,54 @@
+[id='securing-builds-by-strategy']
+= Securing builds by strategy
+include::modules/common-attributes.adoc[]
+:context: securing-builds-by-strategy
+toc::[]
+
+Builds in {product-title} are run in privileged containers that have access to
+the Docker daemon socket. As a security measure, limit who can run builds and
+the strategy that is used for those builds. Custom builds are inherently less
+safe than Source builds, given that they can execute any code in the build with
+potentially full access to the node's Docker socket, and as such are disabled by
+default. Docker build permissions should also be granted with caution, as a
+vulnerability in the Docker build logic could result in a privileges being
+granted on the host node.
```

Comment:
all references to the docker socket should be removed from this paragraph.

```
Builds in {product-title} are run in privileged containers. Depending on the build strategy used, this can allow a user who can run builds to escalate their permissions on the cluster and host nodes.  As a security measure, limit who can run builds and
the strategy that is used for those builds. Custom builds are inherently less
safe than Source builds, given that they can execute any code within a privileged container, and as such are disabled by
default. Docker build permissions should also be granted with caution, as a
vulnerability in the Dockerfile processing logic could result in a privileges being
granted on the host node.
```